### DISCUSSION TITLE
Hiding `Pages` submenu on wagtail

### DISCUSSION THREAD

**Discussion by wes-otf** [OpenTechFund/opentech.fund#4115] [General]
Is the pages submenu of any use anymore? Can we hide it so users don't accidentally try to edit any of the old pages on there? seems like it'd be one less thing to trip up staff - we just had a user that was causing backend errors by trying to view/edit those pages

**Comment by wes-otf** [comment]
![Screenshot 2024-08-28 at 16 54 52](https://github.com/user-attachments/assets/39da9cea-0c60-4d7a-9b6f-a2c9ae025751)

**Comment by frjo** [comment]
The front page and each fund/lab/round is technically a wagtail page. The only way to edit the intro on the front page is via "Pages".

This should be rebuilt so "Pages" can be hidden.

**Comment by frjo** [comment]
First step here https://github.com/HyphaApp/hypha/issues/4116

**Comment by wes-otf** [comment]
closed via #4127
